screens/CollectionContacts.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react';
+import { View, Text, StyleSheet, FlatList } from 'react-native';
+import { useQuery } from '@apollo/client';
+
+import Loading from '../components/ui/Loading';
+import { COLORS, SIZES } from '../constants';
+import { GET_COLLECTION_CONTACTS } from '../graphql/queries/Collection';
+
+interface Props {
+  route: {
+    params: {
+      id: string;
+    };
+  };
+}
+interface ContactProp {
+  id: string;
+  name: string;
+  phone: string;
+}
+
+const CollectionContacts = ({ route }: Props) => {
+  const { id: collectionId } = route.params;
+  const [contactList, setContactList] = useState<ContactProp[]>([]);
+
+  const { loading } = useQuery(GET_COLLECTION_CONTACTS, {
+    variables: { id: collectionId },
+    nextFetchPolicy: 'network-only',
+    onCompleted: (data) => {
+      const formattedContacts = data.group.group.contacts.map((contact: ContactProp) => {
+        return {
+          id: contact.id,
+          name: contact.name,
+          phone: contact.phone,
+        };
+      });
+      setContactList(formattedContacts);
+    },
```

Comment:
```suggestion

  const { data, loading } = useQuery(GET_COLLECTION_CONTACTS, {
    variables: { id: collectionId },
    nextFetchPolicy: 'network-only',
```

Use data value directly rather than adding to a state.